esmvaltool/preprocessor/_time_area.py

```diff
@@ -161,3 +167,66 @@ def trajectory_cube(mycube, long1, long2, lat1, lat2, plong1, plong2, plat1,
     lon = wspd_subset.coord('longitude').points
     lat = wspd_subset.coord('latitude').points
     return section, lon, lat
+
+
+# set of time axis checks
```

Comment:
The four functions below are almost identical, i.e. contain a lot of duplicated code. Refactor such that there is no duplicated code.

Reply:
I will, in the near future; for now I'll just leave them like this and refactor when I will add the code that will actually use them :)

Reply:
to answer your question down the line: I intend on using these functions (or the aggregate refactored object/function) to check for type of data in multimodel modules when we will deal with other types of data as well (not only monthly data)